src/helpers/registrars.ts

```diff
@@ -94,6 +105,31 @@ function getBlockDefaultHandler(md: MarkdownIt, name: string): DirectiveBlockHan
     };
 }
 
+function buildLeafBlockHandler(config: LeafBlockDirectiveConfig): LeafBlockDirectiveHandler {
```

Comment:
the tests describe the leaf meta, should we put this meta in this function?
```ts
token.meta.leaf = true
```

Reply:
Nope. The developer of directive must explicitly set meta himself if he needs it.